common/changes/@microsoft/rush/simplify-setting-build-rebuild-commands_2019-08-24-02-05.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Update to support simplified build and rebuild settings",
```

Comment:
Since this is a behavioral change that could possibly break previous configurations, we should do a SemVer MINOR bump and describe it more explicitly.  Maybe something like this:

```
"comment": "Improve the command-line.json handling so that the \"rush build\" command can be extended without having to duplicate the built-in options (GitHub #1375)",
```

Reply:
```suggestion
      "comment": "Improve command-line.json handling so that the \"rush build\" and \"rush rebuild\" commands can be extended without having to duplicate the built-in options (GitHub #1375)",
```